cyences_app_for_splunk/default/data/ui/views/cs_linux_reports.xml

```diff
@@ -106,12 +205,98 @@
       </table>
     </panel>
   </row>
+  <row>
+    <panel>
+      <title>Success Login by Host, Users</title>
+      <input type="text" token="tkn_success_login_USERNAME">
+        <label>USERNAME</label>
+        <default>*</default>
+        <initialValue>*</initialValue>
+      </input>
+      <table>
+        <search>
+          <query>`cs_linux_success_login` host=$tkn_host|s$ user IN ($tkn_success_login_USERNAME$)
+| table _time, host, src_ip, user 
+| stats count, values(src_ip) as src_ips by host, user 
+| sort - count</query>
+          <earliest>$timeRange.earliest$</earliest>
+          <latest>$timeRange.latest$</latest>
+        </search>
+        <option name="drilldown">none</option>
+        <option name="refresh.display">progressbar</option>
+      </table>
+    </panel>
+  </row>
+  <row>
+    <panel>
+      <title>Failed Login by Host, Users</title>
+      <input type="text" token="tkn_failed_login_USERNAME">
+        <label>USERNAME</label>
+        <default>*</default>
+        <initialValue>*</initialValue>
+      </input>
+      <table>
+        <search>
+          <query>`cs_linux_failed_login` host=$tkn_host|s$ user IN ($tkn_failed_login_USERNAME$)
+| rename src as src_ip 
+| table _time, host, user, src_ip, app
+| stats count, values(src_ip) as src_ips by host, user app
+| sort - count</query>
+          <earliest>$timeRange.earliest$</earliest>
+          <latest>$timeRange.latest$</latest>
+        </search>
+        <option name="drilldown">none</option>
+        <option name="refresh.display">progressbar</option>
+      </table>
+    </panel>
+  </row>
+  <row>
+    <panel>
+      <title>Password Change(unix/linux)</title>
+      <input type="text" token="tkn_password_change_USERNAME">
+        <label>USERNAME</label>
+        <default>*</default>
+        <initialValue>*</initialValue>
+      </input>
+      <event>
+        <search>
+          <query>`cs_linux` "password changed for" host=$tkn_host|s$ user IN ($tkn_password_change_USERNAME$)</query>
+          <earliest>$timeRange.earliest$</earliest>
+          <latest>$timeRange.latest$</latest>
+          <sampleRatio>1</sampleRatio>
+        </search>
+        <option name="count">20</option>
+        <option name="list.drilldown">none</option>
+        <option name="list.wrap">1</option>
```

Comment:
Let's recheck options as .41 it does not display any results.

Reply:
I have checked queries in Ethihad bank env. and it is working as expected.